00XX:reassert-fee-payment-method

```diff
@@ -0,0 +1,54 @@
+## HIP: Driving Value Into The IOT Network Through IOT Token Payments
+- Author: @maxgold91 
+- Start Date: 2023-04-17
+- Category: Technical
+- Status: Draft
+- Original PR:
+- Tracking Issue:
+- Vote Requirements: 
+
+## Summary
+
+This HIP proposes changing the method of payment for Location Assert Fees from HNT tokens to IOT tokens. The primary purpose of this change is to drive value into IOT tokens and provide them with a small amount of utility. This HIP does not propose altering any other aspect of Location Assert Fees. The proposed changes are intended to be implemented in compliance with the rules set forth in [HIP-51](https://github.com/helium/HIP/blob/main/0051-helium-dao.md).
```

Comment:
..." proposes changing the denomination of assert and reassert fees from DC to IOT"...

Reply:
I would also be explicit that you are not addressing onboarding fees here and why:
This proposal deliberately does not address Onboarding Fees, as we find the tokenomics within that fee relationship are .... (whatever you think)